Library/Formula/cmake.rb

```diff
@@ -34,7 +34,10 @@ class Cmake < Formula
   option "without-docs", "Don't build man pages"
   depends_on :python => :build if MacOS.version <= :snow_leopard && build.with?("docs")
   depends_on "xz" # For LZMA
-  depends_on "qt" => :optional
+
+  # The GUI option previously offered with `--with-qt` is no longer offered due to
+  # potential future failure and misery. For the GUI option, please tap `caskroom/cask`
+  # and do `brew cask install cmake`. Thanks!
```

Comment:
"The `with-qt` GUI option was removed due to circular dependencies if CMake is build with Qt support and Qt is built with MySQL support as MySQL uses CMake. For the GUI application please instead use `brew install caskroom/cask/cmake`.